plugins/inputs/ctrlx_datalayer/README.md

```diff
@@ -0,0 +1,385 @@
+      ## Only used internally to detect broken network connections.
+      # keep_alive_interval = "60s"
+
+      ## The interval an "error" message is sent if an error was received from a node. (default: 10s)
+      ## Higher values reduce load on output target and network in case of errors by limiting frequency of error messages.
+      # error_interval = "20s"
+
+      ## The interval that defines the fastest rate at which the node values should be sampled and values captured. (default: 1s)
+      ## The sampling frequency should be adjusted to the dynamics of the signal to be sampled.
+      ## Higher sampling frequence increases load on ctrlX Data Layer.
+      ## The sampling frequency can be higher, than the publish interval. Captured samples are put in a queue and sent in publish interval.
+      ## Note: The minimum sampling interval can be overruled by a global setting in the ctrlX Data Layer configuration ('datalayer/subscriptions/settings').
+      # sampling_interval = "1s"
+
+      ## The requested size of the node value queue. (default: 10)
+      ## Relevant if more values are captured than can be sent.
+      # queue_size = 10
+
+      ## The behaviour of the queue if it is full. (default: "DiscardOldest")
+      ## Possible values: 
+      ## - "DiscardOldest"
+      ##   The oldest value gets deleted from the queue when it is full.
+      ## - "DiscardNewest"
+      ##   The newest value gets deleted from the queue when it is full.
+      # queue_behaviour = "DiscardOldest"
+
+      ## The filter when a new value will be sampled. (default: 0.0)
+      ## Calculation rule: If (abs(lastCapturedValue - newValue) > dead_band_value) capture(newValue).
+      # dead_band_value = 0.0
+
+      ## The conditions on which a sample should be captured and thus will be sent as a message. (default: "StatusValue")
+      ## Possible values:
+      ## - "Status"
+      ##   Capture the value only, when the state of the node changes from or to error state. Value changes are ignored.
+      ## - "StatusValue" 
+      ##   Capture when the value changes or the node changes from or to error state.
+      ##   See also 'dead_band_value' for what is considered as a value change.
+      ## - "StatusValueTimestamp": 
+      ##   Capture even if the value is the same, but the timestamp of the value is newer.
+      ##   Note: This might lead to high load on the network because every sample will be sent as a message
+      ##   even if the value of the node did not change.
+      # value_change = "StatusValue"
+      
+```
+
+## Metrics
+
+All measurements are tagged with the hostname of the device and the
+corresponding node address as defined in the ctrlX Data Layer.
+
+- measurement name
+  - tags:
+    - `ctrlx_host` (ctrlX host where the metrics are gathered from)
+    - `host` (The host where the telegraf agent is running)
+    - `node` (Address of the ctrlX Data Layer node)
```

Comment:
I don't think you have `host` anymore and `node` should be `source` as per unwritten convention ;-), so can you please verify this section!?